src/NUnitFramework/framework/Constraints/Operators/ExactCountOperator.cs

```diff
@@ -39,17 +39,24 @@ public class ExactCountOperator : CollectionOperator
         /// <param name="expectedCount">The expected count</param>
         public ExactCountOperator(int expectedCount)
         {
+            // Collection Operators stack on everything
+            // and allow all other ops to stack on them
+            this.left_precedence = 1;
+            this.right_precedence = 10;
+
```

Comment:
This is copied from the former base constructor in CollectionOperator. I don't particularly like this.